### PR TITLE
add sample for using single library in README

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
Just a start. Open to suggestion what to add/change.

**Comment by weidongxu-microsoft** [comment]
https://github.com/Azure/azure-libraries-for-java/issues/869

File: README.md [hunk 1]
```diff
@@ -603,6 +603,41 @@ SqlDatabase database = sqlServer.databases().define("myNewDatabase")
...
 
+### Use single library
+
+For instance, if you only need azure-mgmt-appservice library from 1.27.2, and wish to limit the dependencies, using following dependency instead in POM.
+
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-mgmt-appservice</artifactId>
+    <version>1.27.2</version>
```
Comment (yaohaizh) [inline README.md]: Update to use the latest vresion

**Comment by weidongxu-microsoft** [comment]
Updated automation tool as well.

https://github.com/weidongxu-microsoft/release-automation

**Pull request merged**